File: spec/code_samples/C#/payments@{id}/get.cs
var api = CheckoutApi.Create("your secret key");
var threeDsSessionId = "sid_y3oqhf46pyzuxjbcn2giaqnb44";
GetPaymentResponse payment = await api.Payments.GetAsync(threeDsSessionId);

if (payment.Approved)
{            
    var cardSourceId = payment.Source.AsCard().Id;
}

File: spec/code_samples/PHP/payments@{id}/get.php
<?php

$checkout = new CheckoutApi('your secret key');
$threeDsSessionId = 'sid_y3oqhf46pyzuxjbcn2giaqnb44';

$details = $checkout->payments()->details($threeDsSessionId);

if($details->isSuccessful()) {
    return $details->getSourceId();
}
